app/_hub/kong-inc/rate-limiting/2.1-x.md

```diff
@@ -0,0 +1,233 @@
+---
+name: Rate Limiting
+publisher: Kong Inc.
+redirect_from:
+  - /enterprise/0.35-x/rate-limiting/
+version: 2.1.0
+
+desc: Rate-limit how many HTTP requests a developer can make
+description: |
+  Rate limit how many HTTP requests a developer can make in a given period of seconds, minutes, hours, days, months or years. If the underlying Service/Route (or deprecated API entity) has no authentication layer, the **Client IP** address will be used, otherwise the Consumer will be used if an authentication plugin has been configured.
+
+  <div class="alert alert-warning">
+    <strong>Note:</strong> The functionality of this plugin as bundled
+    with versions of Kong prior to 0.13.1 and Kong Enterprise prior to 0.32
+    differs from what is documented herein. Refer to the
+    <a href="https://github.com/Kong/kong/blob/master/CHANGELOG.md">CHANGELOG</a>
+    for details.
+  </div>
+
+type: plugin
+categories:
+  - traffic-control
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0.x
```

Comment:
does this need a
`- 2.1.x` entry?

Reply:
And:
`- 2.2.x` entry

Reply:
No, this page is for the 2.1-x version of the plugin. `index.md` is the one I just added for 2.2-x, and it has 2.1.x in the list.